app/trans/http/abs.go

```diff
@@ -0,0 +1,16 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/go-devs-ua/octagon/app/ent"
+)
+
+type UserLogic interface {
+	Signup(ent.User) error
```

Comment:
And here, you can avoid named arguments. It's okay within interface definition.

Reply:
Thanks